_posts/en/newsletters/2021-08-25-newsletter.md

```diff
@@ -0,0 +1,229 @@
+    compensation in some cases, why not accept them in all cases?  The
+    same, though, could be said of proportional fees, and this led to
+    some brief discussion of [upfront fees][topic channel jamming
+    attacks] which could allow nodes to be compensated even for
+    unsuccessful payments.
+
+    Towns also suggested that it is possible for a node to ensure it
+    receives a minimal fee even without a base fee by simply refusing
+    to route payments below a certain size.  For example, a node with
+    a current base fee of 1 sat can ensure it receives at least that
+    much with a 0.1% proportional fee and a minimum amount of 1,000
+    sat.  This would stifle micropayments, but LN nodes already handle
+    small payments without using HTLCs, which eliminates some of the
+    fixed costs and may make purely proportional costs more
+    appropriate, though this remained debated.
+
+    Later in the discussion, Olaoluwa Osuntokun [emphasized][osuntokun
+    post] a point made earlier that there's no clear current need for
+    node operators to change a parameter today for a new pathfinding
+    algorithm that nobody is currently ready to use in production.   He and
+    Corallo want to see if further research and development can allow
+    the algorithm (or a similar one based on different principles) to
+    work nearly equally as well even when base fees are non-zero.
+
+    No clear conclusion to the discussion was reached as of this
+    writing.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why does Bitcoin Core support a transaction index but not an address index?]({{bse}}107619)
+  Andrew Chow explains the historical significance of Bitcoin Core's transaction
+  index in validating transactions (since [replaced by the UTXO database]
+  [bitcoin core #1677]) and outlines objections to adding an address
+  index feature to Bitcoin Core including maintenance overhead, lack of compelling use
+  cases, and concerns that such an index is outside of Bitcoin Core's project scope.
+
+- [Is the `mempool` P2P message reliable?]({{bse}}108229)
+  Claris and Pieter Wuille explain the history of the [`mempool`][mempool
+  message] P2P message. After being introduced in [BIP35][] to expose a node's
+  mempool contents to SPV peers, the message was then moved to only be
+  accessible with [BIP37][]'s [bloom filters][topic transaction bloom filtering]
+  which are now [disabled by default][news74 bip37 default].
```

Comment:
Since https://github.com/bitcoin/bitcoin/pull/16248, the `mempool` permission can be configured separately from support for bloom filters. I've left a comment on the stackexchange answer. Perhaps we should also update this summary. Maybe an additional sentence:

Support for `mempool` requests can be configured for individual peers using the `-whitelist=mempool` and `-whitebind=mempool` configuration options.

Reply:
@jnewbery Thanks for this insight and yes Ive added this suggestion now.